frontend/amundsen_application/static/js/features/ColumnList/index.tsx

```diff
@@ -357,7 +354,19 @@ const ColumnList: React.FC<ColumnListProps> = ({
             )}
             <div className="column-name-container">
               <div className="column-name-with-icons">
-                <h3 className="column-name">{title}</h3>
+                {columnDetails.isExpandable ? (
+                  <span
+                    className="column-name-link"
+                    role="button"
+                    tabIndex={0}
+                    onClick={toggleRightPanel.bind(null, columnDetails)}
+                    onKeyDown={toggleRightPanel.bind(null, columnDetails)}
+                  >
+                    <h3 className="column-name">{title}</h3>
+                  </span>
```

Comment:
We usually avoid adding interactivity to non-interactive tags.

I think we would be better to use a text-styled button here, even if we need to do some CSS overrides on top of it.

Reply:
Not sure about the 'onKeyDown' either, what's its purpose?

Reply:
onKeyDown was a a11y requirement for an interactive span, but I changed it to be a button based on your suggestion so it's removed now